gcc/testsuite/rust/compile/unsafe10.rs

```diff
@@ -1,3 +1,6 @@
+#![feature(intrinsics)]
+#![feature(intrinsics)]
```

Comment:
Is the second `feature` attribute on purpose?

Reply:
my bad. i will fix this. :)

Reply:
Thank you so much @TuringKi :)